packages/component-library/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< HEAD
```

Comment:
@jleifeld could you try to resolve this merge conflict? I don't know that the right config should be.